src/main/java/ua/hillel/pages/ItemsPage.java

```diff
@@ -0,0 +1,13 @@
+package ua.hillel.pages;
+
+import static com.codeborne.selenide.Selenide.$;
+
+public class ItemsPage {
+    public void addToCart(String product) {
+        $("#add-to-cart-sauce-labs-fleece-jacket").click();
+    }
```

Comment:
it always adds only the same product.
Please, try to add an ability to select any product by its title, like "Sauce Labs Fleece Jacket", "Sauce Labs Onesie".